CMS/static/scss/reorder.scss

```diff
@@ -1,10 +1,3 @@
-
-#internal-resources {
-  #newest {
-    display: none;
-  }
-}
```

Comment:
I need to investigate something here - one of the delights of government work is that we need to make sure the experience is at a baseline usable even without javascript - it is fine for sort functionality to be missing without javascript but I think getting rid of this will mean that all components are visible all the time

Reply:
my error - good to go - but we should probably do something around hiding those sort buttons if they don't work - something for later